src/dotnet/Program.cs

```diff
@@ -157,6 +160,26 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
 
         }
 
+        private static void ConfigureDotNetForFirstTimeUse()
+        {
+            using (PerfTrace.Current.CaptureTiming())
+            {
+                using (var nugetPackagesArchiver = new NuGetPackagesArchiver())
```

Comment:
This is kind of a lot of stuff to 'new' up on EVERY run of `dotnet` just to check an EnvVar and if a file exists.  Are we sure we want the overhead of all these objects?